packages/govuk-frontend-review/src/views/full-page-examples/feedback/index.mjs

```diff
@@ -1,84 +1,79 @@
+import express from 'express'
 import { body, validationResult } from 'express-validator'
 
 import { formatValidationErrors } from '../../../utils.mjs'
 
-/**
- * @param {import('express').Application} app
- */
-export default (app) => {
-  app.post(
-    '/full-page-examples/feedback',
-    [
-      body('what-were-you-trying-to-do')
```

Comment:
I've removed the square brackets (arrays of middleware) from `express-validator` params

Without them we can remove lots of unnecessary JSDoc since `@types/express` now understand them